README.md

```diff
@@ -86,17 +88,17 @@ Legend:
 ✅ registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.8.1
 ```
 
-## Usage
+### Errors in Output
 
-```md
-completion  : Generate the autocompletion script for the specified shell
-help        : Help about any command
-images      : Check which images in your cluster support arm64
-```
+At present the tool will show an error if it cannot connect to the repository for an image for any reason.
+This could include:
+* Authentication failure
+* Communication error
+* Image no longer available
 
 ## Private Registry Authentication
 
-`kubearchinspect` uses the credential helper defined in `~/.docker/config.json` for authenticating with private registries.
+When `kubearchinspect` discovers an image from a registry that requires authentication, it uses the credential helper defined in `~/.docker/config.json` to obtain the required credentials. This is the same as the `docker` command line.
 
 ## Releases
```

Comment:
I'm not sure what a product release is in this context, is it that pre-releases such as betas don't have a release note? Or would this be in the GitHub releases?